pmd-java/src/main/resources/category/java/performance.xml

```diff
@@ -112,6 +112,123 @@ public class Test {
         </example>
     </rule>
 
+    <rule name="AvoidFileInputStream"
+          since="5.8.2"
```

Comment:
it would be 6.0.0 in the best case scenario (closing soon), otherwise it would drop to 6.1.0